back/app/models/phase.rb

```diff
@@ -102,43 +101,48 @@ class Phase < ApplicationRecord
 
   validates :participation_method, inclusion: { in: PARTICIPATION_METHODS }
 
-  # ideation? or voting?
-  with_options if: :can_contain_ideas? do
-    validates :presentation_mode,
-      inclusion: { in: PRESENTATION_MODES }, allow_nil: true
+  with_options if: ->(phase) { phase.method.supports_public_visibility? } do
```

Comment:
I like `supports_public_visibility?`, seems like _the_ most relevant property to define behavior in many places (better than some other properties we are using now).